.github/workflows/run-pytest.yml

```diff
@@ -0,0 +1,66 @@
+name: Run Unit Tests
+
+on:
+  # trigger on pull requests
+  pull_request:
```

Comment:
Do we want this to run on all PRs or ones that target a given branch?

Reply:
All PRs. 🤷‍♂️